mobile/src/main/java/ch/epfl/sdp/mobile/application/Profile.kt

```diff
@@ -35,8 +37,19 @@ interface Profile {
       /** A soft orange value. */
       val Orange = Color("#FFF8A68D")
 
+      /** A soft green value. */
+      val Green200 = Color("#FFA9DBBB")
+
+      /** A green value. */
+      val Green500 = Color("#FF379665")
+
+      /** A purple value. */
+      val Purple = Color("#FF6A5495")
+
       /** A default color. */
       val Default = Orange
+
+      val asList = { listOf(Green200, Green500, Orange, Purple) }
```

Comment:
Nitpick : for consistency with enumerations, what would you think of changing this signature to `values(): Array<Color>` ?

Also, I think you used the Scala syntax here, since (in this code) `asList` is a `() -> List<Color>` and you probably wanted to make it a `List<Color>`. You must remove the enclosing braces.

Reply:
thx, is changed